src/main/scala/supporters/ExpressionTranslator.scala

```diff
@@ -88,10 +89,18 @@ trait ExpressionTranslator {
           }
         )))
 
+        val weight = sourceQuant.info match {
```

Comment:
The same thing needs to happen in the Evaluator: https://github.com/viperproject/silicon/blob/8239c0535fb81b025e1fae39e93df817b7146220/src/main/scala/rules/Evaluator.scala#L631

Reply:
Thanks, I pushed a commit. I was looking for `DefaultEvaluator` as in the comment but I couldn't find that file...